### ISSUE TITLE
Adding file using "Browse Files" button is not working

### ISSUE THREAD

**Issue by Mahmaddz** [tableflowhq/csv-import#228]
I used a React component with a modal. When I click on the "Browse Files" button, it doesn't load the file or work properly.
How to recreate: Just install library in your project and then use the example code provided on docs.

versions:
`csv-import-react: ^1.0.11
react: ^18.3.1`
Component code:
`import { useState } from "react";
import { CSVImporter } from "csv-import-react";

const CsvImportComponent: React.FC = () => {
  const [isOpen, setIsOpen] = useState(false);

  return (
    <>
      <button onClick={() => setIsOpen(true)}>Open CSV Importer</button>

      <CSVImporter
        modalIsOpen={isOpen}
        isModal={true}
        modalOnCloseTriggered={() => setIsOpen(false)}
        // darkMode={true}
        onComplete={(data) => console.log(data)}
        template={{
          columns: [
            {
              name: "First Name",
              key: "first_name",
              required: true,
              description: "The first name of the user",
              suggested_mappings: ["First", "Name"],
            },
            {
              name: "Age",
              data_type: "number",
            },
          ],
        }}
      />
    </>
  );
};

export default CsvImportComponent;`

When i use browse files button it doesnt work. Only drag functionality if working
![SudeepLogisticsPvtLtd-ezgif com-video-to-gif-converter](https://github.com/tableflowhq/csv-import/assets/75247619/f2f30899-86e2-4b17-ac5f-0371165b0c06)
![SudeepLogisticsPvtLtd-ezgif com-video-to-gif-converter (1)](https://github.com/tableflowhq/csv-import/assets/75247619/24b37ced-5198-44fe-a69c-bbc56a24c2de)

**Comment by ansario** [comment]
@Mahmaddz did you figure this out?

**Comment by Mahmaddz** [comment]
> @Mahmaddz did you figure this out?

no, i switched to another library

**Comment by wannabetolkien** [comment]
Hey ! Can I work on this issue?

**Comment by MohammedFaragallah** [comment]
> > @Mahmaddz did you figure this out?
> 
> no, i switched to another library

I'm looking for alternatives can you suggest any?

**Comment by ciminelli** [comment]
> Hey ! Is this issue still open? Can I work on it?

@wannabetolkien Yes! Let me know if there's anything I can help you with to get started. Feel free to make a PR.

**Comment by Mahmaddz** [comment]
> > > [@Mahmaddz](https://github.com/Mahmaddz) did you figure this out?
> > 
> > 
> > no, i switched to another library
> 
> I'm looking for alternatives can you suggest any?

I used this `react-csv-importer`
